tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -83,6 +83,21 @@ Configure the bucket access for TiDB Cloud and get the Role ARN as follows:
 
             For example, `"Resource": "arn:aws:s3:::tidb-cloud-source-data"`.
 
```

Comment:
The following part is copied from https://docs.pingcap.com/tidbcloud/troubleshoot-import-access-denied-error#check-the-policy-of-the-iam-role